sources/packages/backend/apps/db-migrations/src/sql/Types/Add-application-offering-change-assessment-trigger-type.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TYPE sims.assessment_trigger_types
+ADD
+  VALUE if not exists 'Application offering change';
```

Comment:
Should we consider using AFTER or BEFORE an existing enum value?

Reply:
for the assessment trigger type, I don't think it matters. correct me if Iam wrong @andrewsignori-aot @JasonCTang @guru-aot @andrepestana-aot

Reply:
My understanding is that in the way it is it will be just appended to the end. I do not see a problem with it.

Reply:
Got it.  It was off my memory that it appends to the end. I am good 😊